### PR TITLE
debtor birthdate fixes and fee hint updates

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#8583

*Description of changes:*
- allow blank of birthdate
- some fee hint updates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks good as usual.

File: ppr-ui/src/views/newRegistration/LengthTrust.vue [hunk 1]
```diff
@@ -131,16 +138,38 @@ export default class LengthTrust extends Vue {
                 'Please note that this must be within the last 21 days. The length of the Lien is automatically set ' +
                 'to 180 days.'
       case APIRegistrationTypes.MARRIAGE_MH:
+        return (
```
Comment (doug-lovett) [inline ppr-ui/src/views/newRegistration/LengthTrust.vue:142]: Steph, can you add a change to the repairer's lien text to add a space after the end of the first sentence: change '...surrendered.' +
to
'...surrendered. ' +
This is the last ux assurance change for RL.

Comment (jordiwes) [reply ppr-ui/src/views/newRegistration/LengthTrust.vue:142]: done

Comment (doug-lovett) [reply ppr-ui/src/views/newRegistration/LengthTrust.vue:142]: Thanks.

**Review by kialj876 — APPROVED** [review]

**Pull request merged**